lib/peatio/config.rb

```diff
@@ -0,0 +1,15 @@
+module Peatio
+  class Config < OpenStruct
+
+    def method_missing(sym, *args, &blk)
```

Comment:
Why so magic? Prefer to have config structure with strictly defined fields along with required and optional values specified as Hash.

Right now when developer look at Config he has no idea about which configuration values are actually used by gem.